helm/up9/templates/up9-limit-range.yaml

```diff
@@ -5,7 +5,11 @@ metadata:
   namespace: {{ .Values.namespace }}
 spec:
   limits:
-    - default:
+    - type: Container
+      default:
         memory: "512Mi"
-        cpu: "200m"
-      type: Container
+        cpu: "500m"
+      defaultRequest:
+        memory: "100Mi"
+        cpu: "100m"
```

Comment:
those values are per namespace it the deployment/pod/job/etc configure it for themselves. 
it will be used for example for test runs (when we are not configuring limits).